x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/reindex/flyout/progress.test.tsx

```diff
@@ -118,7 +118,6 @@ describe('ReindexProgress', () => {
           {
             lastCompletedStep: ReindexStep.created,
             status: ReindexStatus.inProgress,
-            indexGroup: IndexGroup.ml,
```

Comment:
This test is described as "adds steps for index groups". Should we just remove the entire test?

Reply:
Good call!